shared/src/commonMain/kotlin/org/hyperskill/app/profile/presentation/ProfileFeature.kt

```diff
@@ -51,7 +51,10 @@ interface ProfileFeature {
         ) : StreakFreezeState
 
         @Serializable
-        data class NotEnoughGems(val price: Int) : StreakFreezeState
+        data class NotEnoughGems(
+            val streakFreezeProductId: Long,
```

Comment:
Where `streakFreezeProductId` is used?

Reply:
OK, found `ProfileFeature.StreakFreezeState.CanBuy(state.streakFreezeState.streakFreezeProductId, state.streakFreezeState.price)`